cancermuts/datasources.py

```diff
@@ -534,11 +534,8 @@ def __init__(self, database_files=None, database_encoding=None):
         dataframes = []
 
         if database_files is None:
-            database_dir   = '/data/databases/cosmic'
-            databases      = [  'CosmicMutantExport' ]
-            self._database_files = [os.path.join(database_dir, i)+'.tsv' for i in databases]
-        else:
-            self._database_files = database_files
+            self.log.error('The database file for COSMIC is not specified.')
+            raise FileNotFoundError('The database file for COSMIC is not specified.')
```

Comment:
now that I see this, I think it would be a better idea to just make the `database_files` argument mandatory (i.e. without a default argument). This way we wouldn't even have to check and would just use a basic feature of the language :)). I think we should turn this check into checking if there is at least one element in the list